src/Popover/elements.js

```diff
@@ -100,4 +99,5 @@ export const PopoverContentWrapper = styled.div`
 
 export const PopoverTriggerWrapper = styled.span`
   display: inline-block;
+  width: auto;
```

Comment:
you should use `inherit` instead of `auto` because if you set a width a the parent component we need inherit from it